LibraryApp/src/components/form.js

```diff
@@ -68,9 +60,37 @@ class Form extends React.PureComponent {
     this.resetAllFields = this.resetAllFields.bind(this);
   }
 
-  displayBooksList() {
-    const {setActiveTab} = this.props;
-    setActiveTab(booksList);
+  componentDidMount() {
```

Comment:
use .android.js

Reply:
Created both .android.js (for Android Devices) and .ios.js (for IOS Devices)